src/Tools/_framework/ToolPanels/GradebookStudent.jsx

```diff
@@ -267,12 +267,16 @@ export default function GradebookStudent() {
   }
 
   let studentName = `${students.contents[userId]?.firstName} ${students.contents[userId]?.lastName}`;
+  const section = students.contents[userId]?.section;
 
   return (
     <>
       <div style={{ marginLeft: "18px" }}>
         <b>Gradebook for {studentName}</b>
       </div>
+      <div style={{ marginLeft: "18px" }}>
```

Comment:
If section is blank, then eliminate this line.

Reply:
Handled section